layouts/css/page-modules/_header.styl

```diff
@@ -74,3 +74,125 @@ header
     border none
     max-height 75px
     max-width 180px
+
+  #hamburger
+    display none
+
+  #Mobile ul 
+    padding 0
+
+  #Mobile ul li 
+    display block
+    list-style none
+
+  #Mobile ul li a 
+    font-size 1.1em
+
+  #Mobile #logo img 
+    margin .5em auto .5em auto
+
+  .menu 
+    text-align center
+    display none
+    background-color #333333
```

Comment:
Please use this(https://github.com/nodejs/nodejs.org/blob/dd12f6d/layouts/css/_variables.styl#L18).